NEWS.md

```diff
@@ -0,0 +1,293 @@
+# 1.7
+
+After a five year hiatus we're back with a GitHub organization, with new admins and new maintainers who have brought a great deal of energy to make a long-awaited and long-needed new release.
+
+Since the last stable release many things have happened:
+
+- jq now lives at https://github.com/jqlang
+- New maintainers, admins, and owners have been recruited.
+- CI, scan builds, release builds etc now use GitHub actions.
+- Lots of documentation improvements and fixes.
+- Web site updated with new auto complete, better section ids for linking, dark mode, etc. @itchyny #2628
+- Release builds for:
+  - Linux `amd64`, `arm64`, `armel`, `armhf`, `i386`, `mips`, `mips64`, `mips64el`, `mips64r6`, `mips64r6el`, `mipsel`, `mipsr6`, `mipsr6el`, `powerpc`, `ppc64el`, `riscv64` and `s390x`
+  - macOS `amd64` and `arm64`
+  - Windows `i386` and  `amd64`
+  - Docker `linux/386`, `linux/amd64`, `linux/arm64`, `linux/mips64le`, `linux/ppc64le`, `linux/riscv64` and `linux/s390x`
+- Docker images are now available from `ghcr.io/jqlang/jq` instead of docker hub.
+- OSS-fuzz.
+
+Full commit log can be found at https://github.com/jqlang/jq/compare/jq-1.6...jq-1.7 but here are some highlights:
+
+## CLI changes
+
+- Make object key color configurable using `JQ_COLORS` environment variable. @itchyny @haguenau @ericpruitt #2703
+  ```sh
+  # this would make "field" yellow (33, the last value)
+  $ JQ_COLORS="1;30:0;37:0;37:0;37:0;32:1;37:1;37:1;33" ./jq -n '{field: 123}'
+  {
+    "field": 123
+  }
+  ```
+- Respect `NO_COLOR` environment variable to disable color output. See https://no-color.org for details. @itchyny #2728
+- Improved `--help` output. Now mentions all options and nicer order. @itchyny #2747 #2766
+- Last output value can now control exit code using `--exit-code`/`-e`. #1697 @ryo1kato
```

Comment:
Could I make a plea for consistency?

```
@itchyny #2728
```
And
```
#1697 @ryo1kato
```

If you used a single style, I could write a pattern to ignore them, and it'd be easier for normal readers to have a single pattern to see...

Reply:
It's suppose to be in one style @user @user... #pr #pr... so is an error, thanks! Was thinking @user #pr @user #pr for a while but sometimes more ppl have been involved then just the PR author so i went with collection of users involed then collection of related PRs